### PR TITLE
Download only featured Subtitles for LegendasTV

### PR DESCRIPTION
**Pull request by dtcabrerizo**
Legendas.TV has a feature to flag subtitles that were uploaded by a known group, adding them toa a featured category. Those subtitles usually have a better quality and some user may prefer to download only featured ones.

### CODE REVIEW

**Comment by morpheus65535** [comment]
Please rebase your PR on development branch. For more information, see https://github.com/morpheus65535/bazarr/blob/master/CONTRIBUTING.md

File: frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bazarr",
-  "version": "1",
+  "version": "1.0.0",
```
Comment (morpheus65535) [inline frontend/package.json:3]: Why do you change this file? It's not required.

**Comment by morpheus65535** [comment]
Other than that it looks good. Just make those changes and I'll merge it. Thanks!

File: frontend/package.json (hunk 1 above)
Comment (dtcabrerizo) [reply frontend/package.json:3]: When running react-script build ti was returning an error because the version wasn't pasreable by semver

Comment (morpheus65535) [reply frontend/package.json:3]: @LASER-Yi what do you think about this one?

Comment (LASER-Yi) [reply frontend/package.json:3]: It’s fine :)

**Pull request merged**